src/App.tsx

```diff
@@ -15,7 +15,7 @@ const RenderDevTool = () => {
   }
   return null;
 };
-// { process.env.NODE_ENV === 'development' ? <ReactQueryDevtools initialIsOpen={false} /> : null }
```

Comment:
why did you remove this

Reply:
I was checking something, that code wasn't neccessay.